docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst

```diff
@@ -74,8 +77,8 @@ Args syntax
      - Name of the event trigger
    * - table
      - true
-     - :ref:`TableName <TableName>`
-     - Name of the table
+     - [ :ref:`TableName <TableName>` | :ref:`QualifiedTable <QualifiedTable>` ]
```

Comment:
As seen here: https://github.com/hasura/graphql-engine/blob/b84db36ebb51acd5b51e1254c103f3097a7c2358/server/src-lib/Hasura/Events/Lib.hs#L97

The schema for table is actually always `QualifiedTable` in the payload. So what we had before here was actually incorrect.

Reply:
So, should i change the documentation again to set QualifiedTable only?

Reply:
Yes, pls. Thanks!